modules/core/src/v2/coins/stx.ts

```diff
@@ -212,11 +213,11 @@ export class Stx extends BaseCoin {
         throw new Error('missing explain tx parameters');
       }
 
-      const factory = accountLib.getBuilder(self.getChain());
+      const factory = accountLib.getBuilder(self.getChain()) as accountLib.Stx.TransactionBuilderFactory;
       const txBuilder = factory.from(txHex);
 
-      if (!(txBuilder instanceof accountLib.BaseCoin.BaseTransactionBuilder)) {
```

Comment:
Why are we removing this check?

Reply:
The factory's type is set to `accountLib.Stx.TransactionBuilderFactory` because it has an additional method to add public keys to the transaction builder (needed for `explainTransaction`), and the base class doesn't have that.